### DISCUSSION TITLE
Master volume to raise/lower all apps

### DISCUSSION THREAD

**Discussion by mschemerii** [kyleneideck/BackgroundMusic#659] [Ideas]
I would like a feature to be added so that I can raise or lower the volume of all apps simultaneously.
This would allow me to silence everything so that I can cherry pick the apps I want sound to be heard from.

In this regard, it might also be nice to have:
1) favorite apps
2) Save the current volume settings.

**Comment by kyleneideck** [comment]
All good ideas. We could put the new volume slider below the System Sounds one and label it "All Apps". That would also be useful for avoiding clipping when you want to make an app louder than it is by default.

If you're interested in working on saving the app volumes, I wrote up some suggestions here: https://github.com/kyleneideck/BackgroundMusic/issues/180#issuecomment-674364878
